File: 9041/test03/uniq_echo.pl
#!/usr/bin/perl -w


%dict=();
foreach my $word (@ARGV) {
	if ($dict{$word}) {
		next;
	} else {
		$dict{$word} = 1;
		push @result, $word;
     	}
	
}
foreach my $word (@result) {
	print "$word ";
}
print"\n";

File: 9041/test05/uniq_echo.py
#!/usr/bin/python3
import sys

out = []
for i in range(1, len(sys.argv)):
    if sys.argv[i] in out:
        continue
    else:
        out.append(sys.argv[i])

for i in range(0, len(out) - 1):
    print(out[i], end=" ")
print(out[len(out) - 1])
        
    
    

    
    
